_projects/2018/poliastro/export-czml.md

```diff
@@ -0,0 +1,72 @@
+---
+name: Export to CZML
+desc: Implement export of Orbit objects to CZML to visualize in Cesium.js
+requirements:
+ - Familiar with the JSON format
+ - Basic experience with JavaScript or TypeScript
+difficulty: medium
+issues:
+ - https://github.com/poliastro/poliastro/issues/117
+ - https://github.com/poliastro/poliastro/issues/287
+mentors:
+ - juanlu001
```

Comment:
same here as above, there need to be at least 2 mentors per projects, and preferably different primary mentors if you plan to accept students for both of the ideas.

Reply:
How many students are we expected to get? If it's more than one I can make an extra effort to find other primary mentors, although I plan to dedicate all my spare time this summer to GSOC (and SOCIS if it happens).

Reply:
it really depends on the applications, but I would advise very strongly planning to take on more than one primary mentorship, and even then every project idea has to have at least one backup mentors.

Reply:
Fixed!